### PR TITLE
[G&M] Trigger extraction for each mapping

### PR DESCRIPTION
**Pull request by smmr-dn**
Added function to trigger extraction for each mapping. User can trigger this functionality by utilizing the submenu item "Run extraction"
run_extraction

### CODE REVIEW

File: packages/itwin/grouping-mapping-widget/src/components/Mappings/MappingViewActionGroup.tsx [hunk 1]
```diff
@@ -21,7 +22,9 @@ interface MappingUIActionGroupProps {
   onToggleExtraction: (mapping: Mapping) => Promise<void>;
   onRefresh: () => Promise<void>;
   onClickMappingModify?: (mapping: Mapping) => void;
+  onRefreshExtractionStatus: () => Promise<void>;
```
Comment (Guillar1) [inline packages/itwin/grouping-mapping-widget/src/components/Mappings/MappingViewActionGroup.tsx:25]: Doesn't seem used or needed.

File: packages/itwin/grouping-mapping-widget/src/components/Mappings/hooks/useMappingsOperations.ts [hunk 2]
```diff
@@ -50,7 +50,24 @@ export const useMappingsOperations = ({ iModelId, getAccessToken, mappingClient
...
+      const extractionRequest: ExtractionRunRequest | undefined = {
+        mappings: mapping === undefined ? undefined : [
+          {
+            id: mapping.id,
+          },
+        ],
+      };
+      await extractionClient.runExtraction(accessToken, iModelId, extractionRequest);
+    },
+    onSuccess: async () => {
```
Comment (Guillar1) [inline packages/itwin/grouping-mapping-widget/src/components/Mappings/hooks/useMappingsOperations.ts:65]: You can pass this callback in directly.

Comment (smmr-dn) [reply packages/itwin/grouping-mapping-widget/src/components/Mappings/hooks/useMappingsOperations.ts]: I have made the change.

File: packages/itwin/grouping-mapping-widget/src/components/Mappings/MappingViewActionGroup.tsx (hunk 1 above)
Comment (smmr-dn) [reply packages/itwin/grouping-mapping-widget/src/components/Mappings/MappingViewActionGroup.tsx]: I have removed the prop.

**Review by Guillar1 — APPROVED** [review]

**Comment by Guillar1** [comment]
@aruniverse can you approve?

**Review by aruniverse — APPROVED** [review]

**Comment by smmr-dn** [comment]
Not needed anymore

**Pull request closed without merging**